### ISSUE TITLE
No reading validation file

### ISSUE THREAD

**Issue by aguirrejuan** [weecology/DeepForest#557]
Hi, 

Thank you very much for the package. 

I have noticed that the callback related to the validation file is not loaded. This can be due to the logic in the next line
https://github.com/weecology/DeepForest/blob/8144face5ed34509e1872aa482408437091da120/deepforest/main.py#L172C8-L172C62

I presume that the code must be 

`if  self.config["validation"]["csv_file"] is not None:`

Thanks very much.

**Comment by ethanwhite** [comment]
Thanks for this issue as well @aguirrejuan. Can your provide either a reproducible example or more description to help us understand when this isn't happening when you expect it to?

**Comment by aguirrejuan** [comment]
Hello, thank you for your prompt reply. I noticed that the validation file is currently loading.

However, when I launch the model, the following message appears:

"No validation file provided. Turning off validation loop."

I realized that this issue occurs because the training creation is initiated in the __init__ method before the configuration settings are added to the dictionary. But, once I make the call with the correct configuration, the expected behavior is achieved.

Thank you for your assistance.
